packages/patternfly-4/react-core/src/components/OptionsMenu/OptionsMenuItem.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import { css, getModifier } from '@patternfly/react-styles';
+import styles from '@patternfly/patternfly/components/OptionsMenu/options-menu.css';
+import { CheckIcon } from '@patternfly/react-icons';
+import { KEY_CODES } from '../../helpers';
+import { Omit } from '../../helpers/typeUtils';
+
+export interface OptionsMenuItemProps extends Omit<React.HTMLProps<HTMLButtonElement>, 'onSelect' | 'onClick' | 'onKeyDown' | 'type'>{
+  /** Anything which can be rendered as an Options menu item */
+  children?: React.ReactNode;
+  /** Classes applied to root element of an Options menu item */
+  className?: string;
+  /** Render Options menu item as selected */
+  isSelected?: boolean;
+  /** Render Options menu item as disabled option */
+  isDisabled?: boolean;
+  /** Callback for when this Options menu item is selected */
+  onSelect?(event: React.MouseEvent<HTMLButtonElement>|React.KeyboardEvent): void;
```

Comment:
Change to and arrow function, `onSelect?: (event: React.MouseEvent<HTMLButtonElement>|React.KeyboardEvent) => void`